daemon/ZilliqaDaemon.h

```diff
@@ -62,9 +64,9 @@ class ZilliqaDaemon {
   bool DownloadPersistenceFromS3();
```

Comment:
I think we can delete this function, as it has not really been used for a long time

Reply:
We can keep it for now. May be future work for recovery of exchange seed nodes can use it.
It was added for the community seed node but is currently broken.